services/api/src/api/routes/croissant.py

```diff
@@ -199,6 +201,16 @@ def get_croissant_from_dataset_infos(dataset: str, infos: list[Mapping[str, Any]
     }
 
 
+def _get_full_jsonld_parameter(request: Request) -> bool:
+    """Parameter to retrieve the full JSON-LD (full=True) or a truncated/abridged JSON-LD (full=False) with less features."""
+    full_jsonld = get_request_parameter(request, "full", default="false")
+    if full_jsonld.lower() == "true":
+        return True
+    elif full_jsonld.lower() == "false":
+        return False
+    raise MissingRequiredParameterError("Parameter 'full' must have value 'full=true' or 'full=false'")
```

Comment:
I think it's ok to just return False in any other case

```suggestion
    return full_jsonld.lower() == "true"
```

Reply:
Done.